### PR TITLE
Third project

### PR DESCRIPTION
**Pull request by CakeInTech**
- Added a work section.
- Used grid for this portfolio project.
- Used flexbox for the work section.
- Created the about me section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,7 +53,170 @@ <h1>I'm CAkE <br>
             </div>
         </section>
 
+        <section class="works">
```
Comment (Kossi-stack) [inline index.html:58]: - [x] Kindly use a proper margin and padding during styling
- Your Design
alignwell
 - Figma Design
Capturealign

File: index.html [hunk 2]
```diff
@@ -52,7 +53,170 @@ <h1>I'm CAkE <br>
...
 
+
+
+
+
+
+            </div>
+
+        </section>
+        <section class="About">
```
Comment (Kossi-stack) [inline index.html:181]: - [x] Kindly work on the styling check Figma template
- Your Design
Capturesty
- Figma Design
Capturestyl

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @MohamedCK,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

- [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: mobile version PR".
 - [ ] Kindly capitalize the first letter of your commit message in the next milestone so it looks professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 3]
```diff
@@ -52,7 +53,181 @@ <h1>I'm CAkE <br>
...
+                    </ul>
+
+                    <button type="button">Get my resume</button>
+
+
+                    <div class="languages">
+                        <ul>
+                            <li>Languages</li>
+                        </ul>
+                        <hr>
```
Comment (Reem-lab) [inline index.html]: - [x] As the code reviewer asked for customizing the Figma template design as much as possible.👌🏻  so please add the languages like the design and style them.

**Review by Reem-lab — CHANGES_REQUESTED** [review]
## Hi @MohamedCK, 👋🏻👋🏻 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linters error ✅✅
- Good GitHub flow✅✅
- Good commit messages✅✅


 ## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly as the previous code reviewer asked update the style to match the Figma design for the buttons and hover.👌🏻

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @MohamedCK 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**